pkg/marshal/proto_json_marshal.go

```diff
@@ -27,6 +27,8 @@ func DefaultMarshaler() *DefaultMarshal {
 	}
 }
 
+var DefaultMarshaller = NewDefaultMarshaller()
```

Comment:
We try to avoid global variables as much as possible. Here, we can use an init() function to initialise a local variable. Then, we can make the DefaultMarshaller() to return the initialised variable.

Reply:
Alternatively, we can have each struct that uses the `DefaultMarshaller` have its own instance. Is that also acceptable?

Reply:
I initially thought of that but SignPayload and SendHTTPResponse are just functions. Changing signatures of these would requires changes at the SDK too due to the SignPayload function (if I am not wrong). Hence, init() would solve the initial problem.

Reply:
I see your point. Can we avoid the init and simply initialize the local variable statically?

Reply:
Okay. I am fine with that.

Reply:
fixed